trace/log.go

```diff
@@ -43,6 +44,15 @@ func ContextWithLogger(ctx context.Context, l *logrus.Logger) context.Context {
 	return context.WithValue(ctx, logKey{}, le)
 }
 
+// LogTraceInfo logs the given message to the logger obtained from the context and records the message in the trace span.
+//
+// This function does not return any values.
+func LogTraceInfo(ctx context.Context, msg string, fields ...logrus.Fields) {
+	log := LoggerFromContext(ctx).WithFields(mergeFields(fields...))
+	log.WithField("timez", time.Now().Format(time.RFC3339)).Info(msg)
```

Comment:
what's a `timez`? is this a new field?

Reply:
currently when the fields key is "time", it would print `field.time` instead of time. for debugging purposes, i use different key name. i revert it back